app/views/candidate_interface/application_form/submit_success.html.erb

```diff
@@ -14,17 +14,17 @@
   <div class="govuk-grid-column-two-thirds">
     <h2 class="govuk-heading-m">What happens next</h2>
     <p class="govuk-body">
-      You will receive an email from the Apply for postgraduate teacher training service to confirm the successful submission of your application.
+      The Apply for teacher training service will send you an email to confirm the successful submission of your application.
     </p>
 
     <h2 class="govuk-heading-m">References</h2>
-    <p class="govuk-body">We will also contact your referees to ask for your references. When we receive them, we will send them to your training provider. Your provider then has 10 working days to respond to your application.</p>
+    <p class="govuk-body">We will also contact your referees to ask for your references. When we receive them, we will send them to your training provider. Your provider then has 40 working days to respond to your application.</p>
```

Comment:
Yikes. Good catch.

Reply:
@tijmenb Could we derive this number from business rules in code rather than free text?

Reply:
```suggestion
    <p class="govuk-body">We will also contact your referees to ask for your references. When we receive them, we will send them to your training provider. Your provider then has <%= TimeLimitConfig.limits_for(:reject_by_default).first.limit %> working days to respond to your application.</p>
```

Not sure if there is a cleaner way. 🤔

Reply:
Could be calculated in the controller?

```
    def submit_success
      @support_reference = current_application.support_reference
      @days_to_reject_by_default = TimeLimitCalculator.new(rule: :reject_by_default, effective_date: current_application.submitted_at).call
    end
```

Reply:
@duncanjbrown yes

Reply:
Added a card for this so we can merge the present PR https://trello.com/c/8QUESpBF/533-we-hardcode-40-days-when-telling-candidates-about-reject-by-default

Reply:
Filed in https://trello.com/c/8QUESpBF/533-we-hardcode-40-days-when-telling-candidates-about-reject-by-default